webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -102,9 +103,12 @@ def handle_missing(self, date, missing):
 
                 try:
                     MissingProcessedCrash.objects.create(crash_id=crash_id, is_processed=False)
-                except MissingProcessedCrash.IntegrityError:
-                    # If there's already one, that's fine
-                    pass
+                except IntegrityError as ie:
+                    if 'violates unique constraint' in str(ie):
+                        # If there's already one, that's fine
+                        pass
+                    else:
+                        raise
```

Comment:
This block was wrong--`IntegrityError` isn't an attribute of `MissingProcessedCrash`.